monkey/monkey_island/cc/services/authentication_facade/authentication_service.py

```diff
@@ -1,9 +1,9 @@
-from pathlib import Path
-
 from monkey_island.cc.event_queue import IIslandEventQueue, IslandEventTopic
-from monkey_island.cc.models import IslandMode, User
+from monkey_island.cc.models import IslandMode
 from monkey_island.cc.server_utils.encryption import ILockableEncryptor
 
+from .user import User
+
 
 class AuthenticationService:
```

Comment:
This was supposed to be named `AuthenticationFacade`. The directory that contains it should be named `authentication_service`